### DISCUSSION TITLE
Names for our personas

### DISCUSSION THREAD

**Discussion by youngnick** [kubernetes-sigs/gateway-api#2131] [Ideas]
In working on the update for GEP-713 in #2128, I incorporated the parable @kflynn wrote in #2015, and as part of that, I realized that we have standard personas (Intrastructure Admin, Cluster Admin, and Application Developer), but we haven't given those personas names. Giving the personas names is a way to more quickly talk about what each persona needs, and in longer discussions (like the parable now included in #2128), can make them easier to understand.

This discussion is to talk about two things:
- Should we give our personas standard names?
- If so, what should those names be.

I'm proposing that we _should_ give standard names, and that they should be this:
- Infrastructure Admin: Ian, he/him
- Cluster Admin: Charlie, they/them
- Application Developer: Ana, she/her

I've kept to the cryptology-community convention of having the initial letter be the same between the persona and the name. 

I assume that if this is acceptable, the next step will be a small GEP to record this for posterity.

**Comment by kflynn** [comment]
They should 100% have names – without names, they'll tend to get stuck just as abstract descriptions of roles, rather than personas, and devs tend to engage more with personas than abstract roles.

As far as _which_ names, yeah, I'm OK with Ana, Charlie, and Ian (even given my long history with Jane). Happy to help with the GEP, and rest assured that I'll keep pushing for Ana's wants. 😉
